### ISSUE TITLE
Add in the O.C. report number of times the model is overruled

### ISSUE THREAD

**Issue by dimitriskontosBAY** [Roche/crmPack#617]
**Summary**
- record number of times the model is overruled in the simulations (Uli Beyer request)

**Comment by dimitriskontosBAY** [comment]
Could we clarrify what is meant by ''overruled''?
Perphaps give a simple example?

**Comment by danielinteractive** [comment]
Yeah I think we will need more discussion here. Main idea is that the `NextBest` rules (e.g. NCRM) have a `doselimit` and the question would e.g. be how often just this limit is returned as the `nextBest` recommendation. Meaning that with a lower dose limit coming from increment rules the model alone recommendation would have been higher
